src/FileSystem-Tests-Core/FileLocatorTest.class.st

```diff
@@ -259,6 +259,13 @@ FileLocatorTest >> testMoveTo [
 	]
 ]
 
+{ #category : #'resolution tests' }
+FileLocatorTest >> testMusicDirectory [
+
+	locator := FileLocator music.
+	self assert: locator resolve equals: FileLocator music resolve
```

Comment:
What is the purpose of this test?

It's testing that x resolve = x resolve...?
The assertion does not make sense to me.

Maybe just a smoke test is enough?